wallettool/src/main/graal/resource-config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "resources": {
+    "includes": [
+      {"pattern": ".*/english.txt$"}
+    ]
+  }
+}
```

Comment:
Missing EOL here, is that intentional? If this file is generated, then the missing EOL is ok (should perhaps be reported as a bug to the generator).

Reply:
I actually used copy/paste to make this file, so it was an error on my part. I'll fix it.